laceworksdk/http_session.py

```diff
@@ -220,115 +248,141 @@ def get(self, uri, org=False):
 
         return response
 
-    def patch(self, uri, org=False, data=None, param=None):
+    def get(self, uri, params=None, **kwargs):
         """
-        :param uri: uri to send the HTTP POST request to
-        :param org: boolean representing whether the request should be performed at the Organization level
-        :param data: json object containing the data
-        :param param: python object containing the parameters
+        :param uri: uri to send the HTTP GET request to
+        :param params: dict of parameters for the HTTP request
+        :param kwargs: passed on to the requests package
 
         :return: response json
 
-        :raises: ApiError if unable to get a connection
+        :raises: ApiError if anything but expected response code is returned
         """
 
-        self._check_access_token()
+        # Perform a GET request
+        response = self._request("GET", uri, params=params, **kwargs)
 
-        uri = f"{self._base_url}{uri}"
+        return response
 
-        logger.info(f"PATCH request to URI: {uri}")
-        logger.debug(f"PATCH request data:\n{data}")
+    def get_pages(self, uri, params=None, **kwargs):
```

Comment:
Are there any pagination request settings (i.e. max items per page) here that we would want to promote, or should this simply be part of params?

Reply:
Waiting for final documentation on this, but it looks like max items and payload size limits are "future features": https://docs.google.com/document/d/12z5Q7byMJQEoUFS-eLGspyPN-5eMC3jm/edit#heading=h.dfxjw4abp07x

Reply:
well, we can also add in a `max_items` here and just have a simple counter.. that is not here, but in hte `get_items` function, and then exit the generator once you reach that max number.